pkg/sass_api/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.0.0-beta.22
+
```

Comment:
For consistency, should we add `* No user-visible changes.` here? (or perhaps delete the line from most previous entries)